robottelo/hosts.py

```diff
@@ -2057,6 +2058,10 @@ def register_host_custom_repo(self, module_org, rhel_contenthost, repo_urls):
             # refresh repository metadata on the host
             rhel_contenthost.execute('subscription-manager repos --list')
 
+        # Override the repos to enabled  if needed
+        if override:
+            rhel_contenthost.execute(r'subscription-manager repos --enable \*')
```

Comment:
Can we remove this check for `override` arg, as this helper uses custom repos only which will be disabled every time after 6.14, so it won't be required to check and enable based on arg passed
```suggestion
        # Override the repos to enabled  if needed
        rhel_contenthost.execute(r'subscription-manager repos --enable \*')
```

Reply:
Removed

Reply:
Thanks